src/components/organisms/BalanceGameSection/BalanceGameSection.tsx

```diff
@@ -53,6 +54,56 @@ const BalanceGameSection = ({
 
   const gameStages: GameDetail[] =
     game?.gameDetailResponses ?? gameDefaultDetail;
+  const isGuest = !localStorage.getItem('accessToken');
+
+  const [guestVotedList, setGuestVotedList] = useState<VoteRecord[]>([]);
+
+  useEffect(() => {
+    const updateGuestVotedList = () => {
+      const storedVotes = localStorage.getItem(`game_${gameSetId}`);
+      setGuestVotedList(
+        storedVotes ? (JSON.parse(storedVotes) as VoteRecord[]) : [],
+      );
+    };
+
+    updateGuestVotedList();
+
+    const handleStorageChange = (event: StorageEvent) => {
+      if (event.key === `game_${gameSetId}`) {
+        updateGuestVotedList();
+      }
+    };
+
+    window.addEventListener('storage', handleStorageChange);

```

Comment:
```suggestion
    window.addEventListener('storage', handleStorageChange);
    updateGuestVotedList();
```
와 같이 초기 상태에도 로컬 스토리지를 동기화 하는건 어떨까요?

Reply:
[d8e2a13](https://github.com/JECT-Study/PICK-O-Client/pull/216/commits/d8e2a137cc1ff04a2fa2932fd612016a647090f6)